tests/lib/tools/tests.invariant

```diff
@@ -104,6 +104,18 @@ check_stray_dbus_daemon() {
 	fi
 }
 
+check_leftover_defer_sh() {
+	n="$1" # invariant name
+	(
+		find "$PROJECT_PATH" -name defer.sh > "/tmp/tests.invariant.$n"
+	) > "/tmp/tests.invariant.$n"
```

Comment:
Could a reboot afect what we have in /tmp ?

Reply:
It could but I don't see how that is relevant. The temporary file is only used briefly and does not need to be retained after this function returns.